util/constants.ts

```diff
@@ -1,7 +1,7 @@
 export const GITHUB_REPO_URL = 'https://github.com/smlxl/evm.codes'
 
 // Currently active hardfork from the ones available:
-// See: https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/common/src/hardforks
+// See: https://github.com/ethereumjs/ethereumjs-monorepo/tree/stateless/packages/common/src/hardforks
```

Comment:
That isn't the main branch though - I think you mean https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/common/src/hardforks.ts ?